src/shared/components/FormComponents/TagsController/TagsController.styles.ts

```diff
@@ -14,22 +14,24 @@ export const StyledTextField = styled(TextField, shouldForwardProp)`
     }
   }
 
-  ${({
-    isSecondaryUiType,
-    showInputLabel,
-    setMinWidth,
-  }: {
-    isSecondaryUiType: boolean;
-    showInputLabel: boolean;
-    setMinWidth: boolean;
-  }) =>
+  input::placeholder {
+    color: ${variables.palette.outline};
+    opacity: 1;
+  }
+
+  ${({ isSecondaryUiType, setMinWidth }: { isSecondaryUiType: boolean; setMinWidth: boolean }) =>
     isSecondaryUiType &&
     `
     .MuiInputBase-root {
       min-height: 5.6rem;
-      padding: ${theme.spacing(0.4, 1, 0.4, 4)};
+      padding: ${theme.spacing(0, 1, 0, 4)};
       display: flex;
-      flex-wrap: ${showInputLabel ? 'unset' : 'wrap'};
```

Comment:
It seems this will cause braking the layout, if there are more than two rows:
before:
![image](https://github.com/user-attachments/assets/7f96f640-f006-4276-b142-5d88abc5302f)
after:
![image](https://github.com/user-attachments/assets/c22e67bb-18db-4afa-8c29-2417d7d3b993)

Reply:
fixed: 
![Снимок экрана 2024-07-30 в 13 02 50](https://github.com/user-attachments/assets/b9bbb7d0-bac4-4b74-8c7a-1804f1e433a7)